scripts/actions/mobskills/walk_the_plank.lua

```diff
@@ -0,0 +1,28 @@
+-----------------------------------
+-- Walk the Plank (Lion)
+-- AoE Damage, Bind, Knockback, dispel
+-- Type: Physical (AoE)
+-- Skillchain Properties: Light/Distortion
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 1
+    local accmod = 2
+    local dmgmod = 6
```

Comment:
6 seems very high for _all_ her moves, no?

Reply:
The damage from all of her moves is *very* conservative. Averaging less than 200 on tough mobs at level 75, and in-line the damage I saw her produce in videos.

Reply:
it would be great if someday before the game shuts down we took the time to figure out the stats and math involved instead of guessing numbers till it looks like a video _looks sideways in the direction of both blue mage and every mobskill **ever**_

AHEM

dmgmod is pretty much used as a multiplier on the weapon damage part of a mobskills damage calculation. if its taking 6 just to get lower end numbers, probably something is not right elsewhere in the skill script.

Reply:
yeah this is whats wrng, and I've foudn teh same mistake waiting to be copy paste din many other skill scripts as well. this is reallya  problem with how function flow is even structioed because we shoudln't expect anyone to knwo or to do this but:

https://github.com/LandSandBoat/server/blob/base/scripts/actions/mobskills/scintillant_lance.lua#L13

`local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getWeaponDmg() * 2.5, `

So in your script her weapon damage isn't even included. its just 6. Your multiplier should probably be less than one after actually including her weapon damage (because its going to be way larger than 6)